ci-operator/step-registry/openstack/conf/createfips/openstack-conf-createfips-ref.yaml

```diff
@@ -7,10 +7,7 @@ ref:
       cpu: 10m
       memory: 100Mi
   env:
-    - name: OS_CLOUD
-      documentation: Name of cloud to use from ${CLUSTER_PROFILE_DIR}/clouds.yaml file
-    - name: OPENSTACK_EXTERNAL_NETWORK
-      documentation: Name of external network to attach FIPs to.
```

Comment:
we should probably keep the variable documentation.

Reply:
woops! thanks